src/main/java/lotto/controller/LottoController.java

```diff
@@ -32,34 +34,47 @@ private Money initMoney() {
             OutputView.printErrorMessage(e);
             return initMoney();
         }
-        OutputView.howMuchBought(money.lottoCount());
         return money;
     }
 
     private LottoTickets initLottoTickets(Money money) {
-        LottoTickets lottoTickets = LottoTicketsService.createLottoTickets(money);
-        OutputView.printTickets(lottoTickets);
-        return lottoTickets;
+        LottoTickets manualLottoTickets = initManualLottoTickets(money);
+        LottoTickets autoLottoTickets = LottoTicketsService.createAutoLottoTickets(money);
+        OutputView.printHowManyTicketsBought(manualLottoTickets.size(), autoLottoTickets.size());
+        LottoTickets mergedLottoTickets = LottoTicketsService.mergeLottoTickets(manualLottoTickets, autoLottoTickets);
```

Comment:
```suggestion
        LottoTickets mergedLottoTickets = LottoTickets.merge(manualLottoTickets, autoLottoTickets);
```
```suggestion
        LottoTickets mergedLottoTickets = manualLottoTickets.merge(autoLottoTickets);
```

기존 코드와 위 두 코드는 어떠한 차이가 있을까요?

Reply:
LottoTickets 에 메세지를 던지도록 구조를 바꾸어, 데이터를 가지는 객체가 일을 하도록 바뀐거 같습니다!
여기서 궁금한 점이, 1번과 같이 static 메서드를 사용할 필요가 있는 것인가? 제 생각에는 자주 쓰이는 메서드가 아니기 때문에 지양해도 될 것 같습니다. 
또 한편으로 드는 생각은, LottoTicketsService에서 새로운 객체를 만들어, merge를 하면 어떨까 하는 것입니다. 아래 두 메서드 포함 기존의 제 메서드 모두 기존 객체의 데이터가 변경되는데, 혹시 프로그램이 커짐에 따라, 두개를 통합한 객체, 수동로또 객체, 자동로또 객체, 이렇게 3개의 객체가 필요로 하는 순간이 올지도 모른다고 생각이 됩니다.
```
1. LottoTickets mergedLottoTickets = LottoTickets.merge(manualLottoTickets, autoLottoTickets);
2. LottoTickets mergedLottoTickets = manualLottoTickets.merge(autoLottoTickets);
```